integration-testing/playwright.config.ts

```diff
@@ -0,0 +1,24 @@
+import * as path from "node:path";
+import { defineConfig, devices } from "@playwright/test";
+import * as dotenv from "dotenv";
+
+dotenv.config({ path: path.resolve(__dirname, ".env") });
+
+export default defineConfig({
+    testDir: "./tests",
+    fullyParallel: true,
+    forbidOnly: !!process.env.CI,
+    retries: process.env.CI ? 2 : 0,
+    workers: 1,
+    reporter: "html",
+    use: {
+        trace: "on-first-retry",
+    },
+
+    projects: [
+        {
+            name: "chromium",
+            use: { ...devices["Desktop Chrome"] },
```

Comment:
I presume this is here because Playwright still relies on a browser for api testing - or can this be omitted?

Reply:
I thought I needed it but it still runs if I remove that entire project block - so I've omitted it now